Appraisals

```diff
@@ -95,7 +95,7 @@ RAILS_VERSIONS = %w(
   4.2.5.1
   4.2.5.2
   4.2.6
-  5.0.0.beta3
+  5.0.0.1
```

Comment:
What should have been tested? So you want to replace 4.2.7.1 with 4.2.7 or add both ? My point is, why you want to test this library against Rails version that should never be used?